src/common/network/NetworkPrivate.hpp

```diff
@@ -60,7 +60,7 @@ class NetworkData
     void emitError(NetworkResult &&result);
     void emitFinally();
 
-    QLatin1String typeString() const;
+    QStringView typeString() const;
```

Comment:
Keep this as a string instead of a string view

Reply:
In case of confusion, I'll note that `QLatin1String` is a string view too. It will be renamed to `QLatin1StringView` in Qt 7.